data/npc/scripts/bank.lua

```diff
@@ -248,6 +254,11 @@ local function creatureSayCallback(cid, type, msg)
 		end
 	elseif npcHandler.topic[cid] == topicList.TRANSFER_PLAYER_WHO then
 		transfer[cid] = getPlayerDatabaseInfo(msg)
+		if not transfer[cid] then
```

Comment:
You may want to add this if-statement to L210 as well as removing the redundant `if transfer[cid] then` in both places.